src/types/defaults.ts

```diff
@@ -127,18 +127,27 @@ type DeepMergeRecordsDefaultHKTInternalPropsToMergeHelper<
 /**
  * Get the value of the property.
  */
-// TODO: Make tail-recursive.
 type DeepMergeRecordsDefaultHKTInternalPropValue<
```

Comment:
@captain-yossarian Think you could help me with converting this type? I'm pretty sure I've got all the other types right but I can't seem to do this one.
I've had a go at it and followed the same pattern that the other type conversions used, but for some reason it doesn't seem to work.